client/landing/stepper/hooks/use-site-migration-status.ts

```diff
@@ -0,0 +1,138 @@
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { useEffect } from 'react';
+import wpcom from 'calypso/lib/wp';
+import type { SitePlugin } from 'calypso/data/plugins/types';
+
+interface Response {
+	plugins: SitePlugin[];
+}
+
+interface SkipStatus {
+	installation: boolean;
+	activation: boolean;
+}
+type SitePluginParam = Pick< SitePlugin, 'slug' | 'name' >;
+
+const fetchPluginsForSite = async ( siteIdOrSlug: string ): Promise< Response > =>
+	wpcom.req.get( `/sites/${ siteIdOrSlug }/plugins?http_envelope=1`, {
+		apiNamespace: 'rest/v1.2',
+	} );
+
+const installPlugin = async ( siteIdOrSlug: string, pluginSlug: string ) =>
+	wpcom.req.post( {
+		path: `/sites/${ siteIdOrSlug }/plugins/${ pluginSlug }/install`,
+		apiNamespace: 'rest/v1.2',
+	} );
+
+const activePlugin = async ( siteIdOrSlug: string, pluginName: string ) =>
```

Comment:
shouldn't this be `activatePlugin`?

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/89805/commits/9a9749cfafda25727ce7c2e06dea49e0b2c72a24